tests/mka/test_mka_stats.py

```diff
@@ -0,0 +1,126 @@
+import pytest
+import time
+
+
+# @pytest.mark.skip(reason="Revisit CI/CD fail")
+def test_mka_stats(api, b2b_raw_config, utils):
+    """
+    Test for the mka configuration
```

Comment:
It will better to add test for field validation. Use any of the config then made `set_config` finally need to validate those field are setting properly within IxNetwork. Please referred this test https://github.com/open-traffic-generator/snappi-ixnetwork/blob/main/tests/bgp/test_bgp_attributes.py#L128